js/kanban.js

```diff
@@ -1461,7 +1461,7 @@
 
          var uniqueID = Math.floor(Math.random() * 999999);
          var formID = "form_add_" + itemtype + "_" + uniqueID;
-         var add_form = "<form id='" + formID + "' class='kanban-add-form kanban-form no-track'>";
+         var add_form = "<form id='" + formID + "' class='kanban-bulk-add-form kanban-form no-track'>";
```

Comment:
I see no occurence of `kanban-bulk-add-form` in current code. Are you sure this does not break some JS binding ?

Reply:
kanban-add-form is used as a selector to preserve new item forms during a column refresh. Without it, bulk item forms are removed during the background refresh.
Perhaps the selector for the event for adding single items needs to be changed instead of this.

Reply:
```preserveNewItemForms``` updated to match with new class ```.kanban-bulk-add-form
```